plugins/in_mqtt/mqtt.c

```diff
@@ -115,6 +116,18 @@ static int in_mqtt_exit(void *data, struct flb_config *config)
     return 0;
 }
 
+/* Configuration properties map */	
+static struct flb_config_map config_map[] = {	
+    {	
+     FLB_CONFIG_MAP_STR, "listen", "0.0.0.0",	
```

Comment:
As far as I can tell, this is incorrect/unnccessary.

Reading the code in flb_output.c" suggests that the parameter "listen" is never
under the control of config_map. So effectively `in_mqtt` should have an empty
config_map definition with no real entry.

* A related changeset here is 8b55bf77aebbd6041d7c9ba48b7f54bff84858fb.

@aSquare14  Can you investigate the code and verify that observation?

Reply:
@fujimotos  Yes, I have rechecked. You are right, a config map is not required for this plugin.

Reply:
I have added an empty config map as discussed